sites/partners/pages/listings/[id]/applications/index.tsx

```diff
@@ -258,7 +300,11 @@ const ApplicationsList = () => {
               <div className="w-56">
                 <Field name="filter-input" register={register} placeholder={t("t.filter")} />
               </div>
-
+              <div className="mt-2">
+                {!validSearch && (
+                  <AlertBox type="notice">Enter at least 3 characters to search</AlertBox>
```

Comment:
This string will need to be added to the partners/page_content/local_overrides/general.json file